src/nickel.rs

```diff
@@ -34,12 +35,21 @@ impl Options {
         self.output_on_listen = output;
         self
     }
+
+    /// The number of threads the server should use or `None` for default.
+    ///
+    /// Defaults to `None`.
```

Comment:
We should probably specify what default means (related to my comment in thread, do we still keep to hypers default or do we maintain our own default)